sklearn/multiclass.py

```diff
@@ -501,18 +504,22 @@ def predict_proba(self, X):
 
     @available_if(_estimators_has("decision_function"))
     def decision_function(self, X):
-        """Returns the distance of each sample from the decision boundary for
-        each class. This can only be used with estimators which implement the
-        decision_function method.
+        """Decision function for the OneVsRestClassifier.
+
+           Return the distance of each sample from the decision boundary for each
+           class. This can only be used with estimators which implement the
+           `decision_function` method.
```

Comment:
```suggestion
        """Decision function for the OneVsRestClassifier.

        Return the distance of each sample from the decision boundary for each
        class. This can only be used with estimators which implement the
        `decision_function` method.
```